include/nzmqt/impl.hpp

```diff
@@ -453,9 +453,6 @@ NZMQT_INLINE void PollingZMQSocket::onMessageReceived(const QList<QByteArray>& m
 
 NZMQT_INLINE PollingZMQContext::PollingZMQContext(QObject* parent_, int io_threads_)
     : super(parent_, io_threads_)
-    , m_pollItemsMutex(QMutex::Recursive)
```

Comment:
Hrmm, sorry about this. Apparently I accidentally included two "commits" worth of changes into one.

The combined commit is still simple enough to review though.